src/shogun/multiclass/tree/CARTree.cpp

```diff
@@ -1496,7 +1498,6 @@ void CCARTree::init()
 	SG_ADD(&m_nominal, "nominal", "feature types");
 	SG_ADD(&m_weights, "weights", "weights");
 	SG_ADD(&m_weights_set, "weights_set", "weights set");
-	SG_ADD(&m_types_set, "types_set", "feature types set");
```

Comment:
this a flag that is set automatically when setting m_nominal, and shouldn't be a property. it also caused some warnings since putting "m_nominal" doesn't update this flag

Reply:
But it should be registered or? Think serialisation

Reply:
yup, good point